crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -1276,8 +1276,8 @@ mod tests {
 
         // pending block has some transactions
         assert!(!starknet.pending_block().get_transactions().is_empty());
-        // blocks collection is empty
-        assert!(starknet.blocks.hash_to_block.is_empty());
+        // blocks collection should not be empty
+        assert!(!starknet.blocks.hash_to_block.is_empty());
```

Comment:
Why not assert length?

Reply:
is it the same thing?

Reply:
Asserting length is better because it will only pass if length is exactly 1. Asserting that something is not empty will pass for any length > 0

Reply:
Sorry, I didn't notice that it's **!**is_empty(), fixed in c371d56e7e619e6dd20c6d4bfb26f6326a90e21c